website/docs/reference/resource-configs/grants.md

```diff
@@ -1,95 +1,67 @@
+resource_types: [models,seeds,snapshots]
+datatype: "string"
 default_value: {}
+id: "grants"
 ---
 
+<Snippet src="available-prerelease-beta-banner" />
+
+You can manage access to the datasets you're producing with dbt by using grants. To implement these permissions, define grants as resource configs on each model, seed, or snapshot. Define the default grants that apply to the entire project in your `dbt_project.yml`, and define model-specific grants within each model's SQL or YAML file.
+
+The grant resource configs enable you to apply permissions to a specific model and set of recipients as soon as the model finishes building.
+
+Use grants with hooks when you have a more advanced problem to solve. For example, if you want to create more granular row- and column-level access, use masking policies, or apply future grants. For more information on hooks, see [Hooks & operations](/building-a-dbt-project/hooks-operations).
+
+You can set grants in `dbt_project.yml` and as a `config` yaml property that applies to the entire dbt project.
+
 <Tabs
   defaultValue="models"
   values={[
     { label: 'Models', value: 'models', },
-    { label: 'Sources', value: 'sources', },
     { label: 'Seeds', value: 'seeds', },
     { label: 'Snapshots', value: 'snapshots', },
-    { label: 'Tests', value: 'tests', },
-    { label: 'Analyses', value: 'analyses', },
-    { label: 'Macros', value: 'macros', },
   ]
 }>
+
 <TabItem value="models">
 
 <File name='models/schema.yml'>
 
 ```yml
-version: 2
-
 models:
-  - name: model_name
+  - name: specific_model
     config:
-      meta: {<dictionary>}
-
-    columns:
-      - name: column_name
-        meta: {<dictionary>}
-
+      grants:
+        select: ['reporter', 'bi']
 ```
 
 </File>
 
-The `meta` config can also be defined:
+The `grants` config can also be defined:
+
 - under the `models` config block in `dbt_project.yml`
 - in a `config()` Jinja macro within a model's SQL file
 
 See [configs and properties](configs-and-properties) for details.
 
 </TabItem>
 
-<TabItem value="sources">
-
-<File name='models/schema.yml'>
-
-```yml
-version: 2
-
-sources:
-  - name: model_name
-    meta: {<dictionary>}
-
-    tables:
-      - name: table_name
-        meta: {<dictionary>}
-
-        columns:
-          - name: column_name
-            meta: {<dictionary>}
-
-```
-
-</File>
-
-</TabItem>
-
 <TabItem value="seeds">
 
 <File name='seeds/schema.yml'>
 
 ```yml
-version: 2
-
 seeds:
   - name: seed_name
     config:
-      meta: {<dictionary>}
-
-    columns:
-      - name: column_name
-        meta: {<dictionary>}
-
+      grants:
+        select: ['reporter', 'bi']
 ```
 
 </File>
 
-The `meta` config can also be defined under the `seeds` config block in `dbt_project.yml`. See [configs and properties](configs-and-properties) for details.
+The `grants` config can also be defined under the `seeds` config block in `dbt_project.yml`. See [configs and properties](configs-and-properties) for details.
 
 </TabItem>
```

Comment:
@jerco I'm not 100% sure I got this example correct

Reply:
This looks right to me!